protobuf/xain_fl/grpc/coordinator.proto

```diff
@@ -55,22 +55,15 @@ message StartTrainingReply {
 
 message EndTrainingRequest {
   ThetaUpdate theta_update = 1;
-  map<string, HistoryValue> history = 2;
-  Metrics metrics = 3;
+  map<string, Metrics> metric = 2;
 
   message ThetaUpdate {
     repeated numproto.protobuf.NDArray theta_prime = 1;
     int32 num_examples = 2;
   }
 
-  /* protobuf doesn't like repeated as map values */
-  message HistoryValue {
-    repeated float values = 1;
-  }
-
   message Metrics {
-    int32 cid = 1;
-    repeated int32 vol_by_class = 2;
+    repeated numproto.protobuf.NDArray metrics = 1;
```

Comment:
Please make sure that you update the proto files in the xain-sdk repo as well.

Reply:
Done, see PR in the xain_sdk repo.